### PR TITLE
sort scoped modules.

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,16 +29,16 @@ const aliased = [
   'utils',
 ];
 
-function isInternalModule({ moduleName }) {
-  return moduleName.startsWith('@qsi/');
+function isScopedModule({ moduleName }) {
```
Comment (taion) [inline index.js:32]: all scoped modules though? isn't there a bunch of public stuff scoped under `@exponent` or whatever?

Comment (jquense) [reply index.js:32]: I figured it probably wouldn't be a big deal, I can't make the stuff configurable, this was sort of the middle ground we chatted about in #2

Comment (taion) [reply index.js:32]: fair enough; fine by me then

Comment (jquense) [reply index.js:32]: I've got a few PR's open on the underlying library so i'm hoping that will give us more options in the future as well.

**Pull request merged**